src/hooks/useIsLegacyAcp/index.js

```diff
@@ -28,7 +28,7 @@ import { hasAcpConfiguration } from "../../accessControl/acp/index";
 export default function useIsLegacyAcp(resourceInfo) {
   const { fetch } = useSession();
   return useSWR(
-    resourceInfo ? acp.getLinkedAcrUrl(resourceInfo) : null,
+    [resourceInfo ? acp.getLinkedAcrUrl(resourceInfo) : null, fetch],
```

Comment:
...although this one returns undefined, instead of null, so this ternary belongs here. I've started a conversation in the SDK team to align on when we return null vs undefined.